doc/content/source/turbulence/auxkernels/kEpsilonViscosity.md

```diff
@@ -0,0 +1,207 @@
+# kEpsilonViscosity
+
+`kEpsilonViscosity` is an auxiliary kernel that computes the **turbulent dynamic viscosity**
+$\mu_t$ used in the k–$\epsilon$ family of turbulence models.
+It forms the closure term for the Reynolds stresses in the momentum equations:
+
+\begin{equation}
+\mu_t = \rho\, C_\mu\, k\, T,
+\end{equation}
+
+where the *turbulent time scale* $T$ and the *effective coefficient* $C_\mu$ depend on the
+selected k–$\epsilon$ variant and the local flow state.
+
+This object implements:
+
+- Standard, Low-Re, Two-Layer, and Realizable viscosity formulations,
+- Several wall treatments (Newton, incremental, linearized, non-equilibrium),
+- Two-layer near-wall blending methods (Wolfstein, Norris–Reynolds, Xu),
```

Comment:
Would be nice to cite something here. Or add al ink to the theory page if it cites these.